.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -155,9 +155,40 @@ jobs:
         name: sycl_lit_${{ inputs.build_artifact_suffix }}
         path: lit.tar.xz
 
-  llvm_test_suite:
+  aws-start-matrix:
+    name: Start AWS Matrix
     needs: build
     if: ${{ inputs.lts_matrix != '' }}
```

Comment:
Sorry for another late comment. Would it be possible to make early AWS steps skip if no aws-type tasks in the matrix?
Basically if no aws-type in the matrix we don't need AWS jobs at all.
That would be very nice optimization at this point.

Otherwise it looks good for merge.

Reply:
Implemented as merge of all AWS start/stop tasks into single job.

Reply:
Implemented aws start/stop skip in case there is no AWS instances specified in the matrix.